ccache.c

```diff
@@ -1898,6 +1957,10 @@ from_cache(enum fromcache_call_mode mode, bool put_object_in_manifest)
 		// The compiler won't generate notes if there is no code
 		get_file_from_cache(cached_cov, output_cov);
 	}
+	if (generating_stackusage && stat(cached_su, &st) == 0 && st.st_size > 0) {
+		// The compiler won't generate notes if there is no code
```

Comment:
I'm wondering if this applies to stackusage the way it did for coverage... (notes)